annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastProcessor.java

```diff
@@ -93,11 +94,31 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
     }
 
     private void buildCreatorClasses(FastEnvironment fastEnvironment) throws IOException {
+        Mustache mustache = mustacheFactory.compile(
+            "com/exactpro/epfast/annotation/processing/FieldSetter.java.mustache");
+        for (FastPackageElement fastPackage : fastEnvironment.getFastPackages()) {
+            for (FastTypeElement fastTypeElement : fastPackage.getFastTypes()) {
+                TypeName typeName = new TypeName(
```

Comment:
I'd move this logic to FastTypeElement. You can store FastPackgeElement there (it can be set in `FastPackageElement.addFastType`) and have just getter which computes the name.